README.md

```diff
@@ -36,9 +36,39 @@ Features
 This gem includes several common features used in 500px service client libraries.
 
 The features are:
+#### Px::Service::Client::Base
```

Comment:
Vertical whitespace (add a newline above this line).

Also I would mention below that the request/multiplexer system uses Typhoeus as the underlying HTTP client to support asynchronicity.